src/main/java/ro/sci/people/Person.java

```diff
@@ -0,0 +1,28 @@
+package main.java.ro.sci.people;
+
+public class Person {
+
+    String first_name;
```

Comment:
Why didn't you use access modifiers?
If the class members need to be visible outside of the class, please specify this with javadoc.
This is breach of encapsulation.